oracle/src/main/java/org/jdbi/v3/oracle/OracleReturning.java

```diff
@@ -56,34 +58,33 @@
  * }
  * </pre>
  * Though you can bind multiple params, and whatnot
+ * <p>
+ * This class is beta, and may be changed incompatibly or removed at any time.
  */
-@Deprecated
-public class OracleReturning<ResultType> implements StatementCustomizer
-{
+@Beta
+public class OracleReturning<ResultType> implements StatementCustomizer {
     private final RowMapper<ResultType> mapper;
     private final List<int[]> binds = new ArrayList<>();
     private StatementContext context;
     private List<ResultType> results;
-    private Class<?> oraclePS ;
-    private Method registerReturnParameter ;
-    private Method getReturnResultSet ;
-    private Object stmt ;
+    private Class<?> oraclePS;
+    private Method registerReturnParameter;
+    private Method getReturnResultSet;
+    private Object stmt;
 
     /**
      * Provide a mapper which knows how to do positional access, sadly the
      * {@link BeanMapper} uses the names in the result set
      *
      * @param mapper Must use only positional access to the result set
      */
-    public OracleReturning(RowMapper<ResultType> mapper)
-    {
+    public OracleReturning(RowMapper<ResultType> mapper) {
         this.mapper = mapper;
         try {
             this.oraclePS = Class.forName("oracle.jdbc.OraclePreparedStatement");
```

Comment:
Do we still need to do all this reflective bullshit now that it's in a separate artifact?